packages/testing/src/api/mockWidgetApi.ts

```diff
@@ -24,11 +24,28 @@ import {
   compareOriginServerTS,
   isValidEventWithRelatesTo,
 } from '@matrix-widget-toolkit/api';
-import { cloneDeep, isEqual, uniqueId } from 'lodash';
 import { Symbols } from 'matrix-widget-api';
 import { NEVER, Subject, concat, filter, from, map, of, takeUntil } from 'rxjs';
 import { Mocked, vi } from 'vitest';
 
+// These 2 functions originally were using lodash. These are replacements for them.
+const uniqueId = (
+  (counter) =>
+  (str = '') =>
+    `${str}${++counter}`
+)(0);
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any -- This is a generic deepEqual function therefore it can accept any type.
```

Comment:
I would argue we should introduce a rule:
- If you can replace lodash with a one-liner: do it
- Else use lodash

Happy do discuss

Reply:
The thing is that lodash is unmaintained since 4 years now and I am not sure if it is great to sit on an ancient dependency.

Reply:
Then I would at least say: Move it to utils and add proper tests…